app/register/components/confirmDialog.tsx

```diff
@@ -35,50 +29,35 @@ export function ConfirmDialog({
     event.currentTarget.innerHTML =
       '<span class="loading loading-ring loading-lg"></span>送信中...'
 
-    const fileName = `${crypto.randomUUID()}.${extention}`
-    const file = await CvtDataUrl2File(imageSrc, fileName)
-    const url = await getServerUrl(fileName)
+    const formElement = document.querySelector('form')
+    const formData = new FormData(formElement as HTMLFormElement)
 
-    await fetch(url, {
-      method: 'PUT',
-      headers: {
-        'Content-Type': fileType,
-        'Content-Disposition': `attachment; filename=${fileName}`,
-      },
-      body: file,
+    fetch('/api/image', {
+      method: 'POST',
+      body: formData,
     })
+      .then((res) => {
+        if (!res.ok) {
+          alert(`画像のアップロードに失敗しました：${res.statusText}`)
+        }
+      })
       .catch((error) => {
-        alert(error)
-        return
+        alert(`画像アップロードエラー：${error}`)
       })
+
+    fetch('/api/user', {
+      method: 'POST',
+      body: formData,
+    })
       .then((res) => {
-        if (!res) {
-          alert('送信に失敗しました')
-        } else if (res.ok) {
-          // values.image[0] からファイル取得困難のためdbdataにs3に保存したファイル名を押し込む。
-          const dbdata = Object.assign(values, { fileName: fileName })
-          fetch('/api/user', {
-            method: 'POST',
-            headers: { 'Content-Type': 'application/json' },
-            body: JSON.stringify(dbdata),
-          })
-            .then((res) => {
-              if (!res.ok) {
-                // DB登録失敗時の画像削除処理は未実装
-                alert(res.statusText)
-                return
-              }
-              router.push('/register/success')
-            })
-            .catch((error) => {
-              // DB登録失敗時の画像削除処理は未実装
-              alert(error)
-            })
-        } else {
-          // DB登録失敗時の画像削除処理は未実装
```

Comment:
It looks this PR doesn't resolve the problem mentioned in this comment,
so I think we shouldn't remain the comment or should make issue.

Reply:
Fetch requests don't save image data if fetch failed.
That's why you don't need to delete image.